docker-compose-api.yml

```
@@ -0,0 +1,16 @@
+version: '3'
+
+services:
+
+  api-server:

```

서버별로 docker-compose 파일이 분리되어있는 이유가 있을까요?! 
컴포즈를 사용하는 이유는 배포를 간편하게 하고 여러 컨테이너를 한번에 관리하기 위해서가 크기 때문에 합치면 좋을 것 같습니다 :)

```suggestion
version: '3'

services:
  api-server:
    image: ${NCP_CONTAINER_REGISTRY_API}/payout-api
    ports:
      - "8080:8080"
    environment:
      DB_HOSTNAME: ${DB_HOSTNAME}
      DB_PORT: ${DB_PORT}
      DB_DATABASE: ${DB_DATABASE}
      DB_USERNAME: ${DB_USERNAME}
      DB_PASSWORD: ${DB_PASSWORD}
      FMP_API_KEY: ${FMP_API_KEY}
    restart: always

  batch:
    image: ${NCP_CONTAINER_REGISTRY_API}/payout-batch
    ports:
      - "8080:8080"
    environment:
      DB_HOSTNAME: ${DB_HOSTNAME}
      DB_PORT: ${DB_PORT}
      DB_DATABASE: ${DB_DATABASE}
      DB_USERNAME: ${DB_USERNAME}
      DB_PASSWORD: ${DB_PASSWORD}
      FMP_API_KEY: ${FMP_API_KEY}
    restart: always


```

---

앗.. 되게 기초적인 실수를 했네요.. 도커 컴포즈를 처음 써봐서 여기저기서 찾아보면서 짰는데 저는 한 yml 파일에 정의되어 있는 컨테이너가 docker compose up 명령어를 실행하면 모두 다 실행되는 것으로 착각했습니다...!! 그래서 api server와 batch server를 분리하기 위해 yml 파일도 나눴어요!
찾아보니 특정 컨테이너만 별도로 실행하는게 가능하군요?! 말씀해주신 내용대로 수정하는게 맞는거 같아요!